### ISSUE TITLE
Cannot tab through commits in history

### ISSUE THREAD

**Issue by guidezpl** [desktop/desktop#14652]
### The problem

Cannot use a11y focus (tabbing) through commits in history.

### Release version

Version 2.9.10 (arm64)

### Operating system

macOS

### Steps to reproduce the behavior

1. Open a repo with history
2. Click 'History'
3. Click a commit
4. Try to tab

Focus goes elsewhere and never comes back

**Comment by sergiou87** [comment]
The problem seems to be that the element focused right before the Commit list is the "Select branch to compare…", which replaces the list of commits with a list of branches. Because of that, if you press <kbd>Tab</kbd> at that point, the list of commits doesn't exist and rotates to the next focusable element.

If you press <kbd>Shift</kbd>+<kbd>Tab</kbd> to rotate backwards, you'll see it gets back to the list of commits.
